x-pack/packages/kbn-elastic-assistant/impl/assistant/conversations/conversation_settings/conversation_settings.tsx

```diff
@@ -152,6 +163,7 @@ export const ConversationSettings: React.FC<ConversationSettingsProps> = React.m
                 ...selectedConversation.apiConfig,
                 connectorId,
                 provider,
+                model: model ?? selectedConversation.apiConfig.model,
```

Comment:
Not intuitive here that `selectedConversation.apiConfig.model` is the fallback case as there are two points of entry here, both selection change, and creation of new connector (which will have a defaultModel) and then selection). Will clarify with comment.

Reply:
Cleaned up and removed `conversation` fallback to make UI more predictable, e.g. when switching from `Azure` or `Preconfigured` to OpenAI, the connector default will be used instead of whatever _last OpenAI connector_ that may've been used for the conversation.